module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AccountMenuTest.php

```diff
@@ -38,6 +38,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  * @retry    4
+ * @skip_html_validation true
```

Comment:
Is there a compelling reason to skip validation here, or is this just a proof of concept? (Just wondering if this was intentionally committed).

Reply:
Oops, that was just the result of committing test change..